ui/ducks/send/send.js

```diff
@@ -1331,17 +1358,24 @@ export function updateSendAmount(amount) {
   };
 }
 
+/**
```

Comment:
This is a type definition block for `TokenDetails`. This was done because `eslint-plugin-jsdoc` didn't seem to understand the deeply-nested parameters (e.g. `@param [payload.details.address]`). I guess it only looks one level deep? Which honestly isn't bad, because at some point it gets confusing to read anyway.

So those three deeply-nested properties were pulled into a type, and that type is referenced below (on line 1378).